### DISCUSSION TITLE
No API endpoint on self-hosted app

### DISCUSSION THREAD

**Discussion by selfagency** [outline/outline#5869] [Self Hosting] [answered]
I am running the latest version bare metal.

Attempting to query my backend:

```hurl
GET https://<my-app-url>/api/documents.list
content-type: application/json
accept: application/json
authorization: Bearer <my-token>
```

Returns:

```json
{"ok":false,"error":"not_found","status":404,"message":"Endpoint not found"}
```

If I try to go to `/api` in my browser, I get a 404 page.

Is there a setting I need to set to switch on API access?

What am I missing?

**Comment by tommoor** [comment, marked as answer]
All API requests (with a couple of exceptions) should be POST not GET
